resources/js/views/settings/Application.vue

```diff
@@ -0,0 +1,471 @@
+<template>
+  <b-container fluid>
+      <h3>
+        {{ $t('settings.application.title') }}
+        <b-button id="application-save-button"
+                  class="float-right"
+                  variant="success"
+                  @click="updateSettings(settings)"
+                  v-if="!viewOnly"
+                  :disabled="isBusy">
+          <span><i class="fas fa-save mr-2"></i>{{ $t('app.save') }}</span>
+        </b-button>
+      </h3>
+
+      <hr>
+
+    <!--Pagination page size settings-->
+    <b-form-group
+      label-for="application-name-input"
+      :description="$t('settings.application.name.description')"
+      :state='fieldState("name")'
+    >
+
+      <template v-slot:label>
+        {{ $t('settings.application.name.title') }}
+      </template>
+
+      <b-form-input id="application-name-input"
+                    v-model="settings.name"
+                    type="text"
+                    :disabled="isBusy || viewOnly"
+                    :state='fieldState("name")'
+      >
+      </b-form-input>
+
+      <template slot='invalid-feedback'>
+        <div v-html="fieldError('name')"></div>
+      </template>
+    </b-form-group>
+
+    <!--Favicon Settings-->
+    <b-form-group
+      label-for="application-favicon-input"
+      :state='(fieldState("favicon") === null && fieldState("favicon_file") === null) ? null : false'
+    >
+
+      <template v-slot:label>
+        {{ $t('settings.application.favicon.title') }}
```

Comment:
Why we need the extra template? The label could be set in the form group through an attribute!